src/App.jsx

```diff
@@ -14,9 +15,9 @@ function App() {
             <Route path='/' element={<Home />} />
             <Route path='/books/claimed' element={<ClaimedBooks />} />
             <Route path='/books/:id' element={<SingleBookDetail />} />
+            <Route path='/addabook' element={<AddBook />} />
```

Comment:
To keep it consistent with your other routes, you should change this to /books/add instead